packages/ui/lib/components/Footer/EmailSubscriptionForm.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react'
+import { Button } from '@unlock-protocol/ui'
+
+export const EMAIL_SUBSCRIPTION_FORM = {
+  portalId: '19942922',
+  formGuid: '868101be-ae3e-422e-bc86-356c96939187',
+}
+
+export function EmailSubscriptionForm() {
+  const { portalId, formGuid } = EMAIL_SUBSCRIPTION_FORM
+  const [email, setEmail] = useState('')
+  const [confirm, setConfirm] = useState(false)
+  const [_, setError] = useState<string | null>(null)
+
+  async function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault()
+    if (!email) {
+      return
+    }
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        formGuid,
+        portalId,
+        fields: [
+          {
+            name: 'email',
+            value: email,
+          },
+        ],
+      }),
+    }
+
+    const endpoint = `https://api.hsforms.com/submissions/v3/integration/submit/${portalId}/${formGuid}`
```

Comment:
We could make the whole endpoint by just a prop... So it does not require the use of Hubspot.